File: C_Programs/ExampleOfFunctionWithReturnTypeAndWithParamaters2.cpp
//function returning a value and with parameters
//create a function calcualte with 2 int parameters and return the sum of 2 no.

#include<stdio.h>
#include<conio.h>

int sum()
{
	int num1,num2,sum;
	printf("Enter 2 numbers : ");
	scanf("%d%d",&num1,&num2);
	sum=num1+num2;
	return sum;
}

int main()
{
	int res;
	printf("\nSum of 2 numbers = %d",res=sum());
	return 0;
}

File: Core_Java_Programs/ExampleOfFunctionWithReturnTypeAndWithParamaters2.java
//function returning a value and with parameters
//create a function calcualte with 2 int parameters and return the sum of 2 no.
import java.util.*;

public class ExampleOfFunctionWithReturnTypeAndWithParamaters2 {
    public static int sum()
    {
        int num1,num2,sum;
        Scanner sc=new Scanner(System.in);

        System.out.println("Enter 2 numbers : ");
        num1=sc.nextInt();
        num2=sc.nextInt();
        sum=num1+num2;
        return sum;
    }

    public static void main(String[] args)
    {
        int res;
        res=sum();
        System.out.println("Sum of 2 numbers = "+res);
    }
}
